src/main/java/uk/gov/hmcts/divorce/divorcecase/model/Application.java

```diff
@@ -467,4 +467,15 @@ public boolean isSolicitorApplication() {
     public boolean isSolicitorPaymentMethodPba() {
         return FEE_PAY_BY_ACCOUNT.equals(this.getSolPaymentHowToPay());
     }
+
+    @JsonIgnore
+    public boolean isApplicant1OffLine() {
```

Comment:
shouldn't these be Applicant.isOffline?

Reply:
Yes, that is what I wanted, but at the moment the only way to work out whether to do off line or not is "applicant1KnowsApplicant2EmailAddress" field in Application.  Moving that field inside the Applicant class, would not work.  We could have a new field inside Applicant that is set on submission, that way we can set the appropriate offline setting for applicant1 and applicant2.